Community/alumni.md

```diff
@@ -324,15 +364,13 @@ Graal produces analysis related to code complexity, quality, dependencies, vulne
 Out of all the five backends provided by Graal, CoCom (Code Complexity) covers a vast majority of the popular languages and CoLic (Code License) supported by NOMOS & ScanCode helps us fetch license & copyright related information from software development repositories and is language independent. Addition of metrics related to these two backends during GSoC period could be applied to a wide range of projects in the future.
 
 
-#### Blog Posts
-
-* [Weekly Posts](https://nishchith.com//20blog19/08/coding-period-3-11.html)
-
-
-#### Social Media
+#### Links
 
+* [Project Link on GSoC Website](https://summerofcode.withgoogle.com/archive/2019/projects/5400999387201536/)
+* [Blog Posts](https://nishchith.com//20blog19/08/coding-period-3-11.html)
 * [LinkedIn](https://www.linkedin.com/in/inishchith/)
 * [Twitter](https://twitter.com/inishchith)

```

Comment:
blog posts link is broken, seems he doesn't maintain the blog anymore
does it make sense to remove it instead of having a broken link in the page?
also, can we remove linkedin and twitter for consistency?